examples/skywalking-tracing/front-envoy-skywalking.yaml

```diff
@@ -0,0 +1,80 @@
+static_resources:
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 8000
+    traffic_direction: OUTBOUND
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          generate_request_id: true
+          tracing:
+            provider:
+              name: envoy.tracers.skywalking
+              typed_config:
+                "@type": type.googleapis.com/envoy.config.trace.v3.SkyWalkingConfig
+                grpc_service:
+                  envoy_grpc:
+                    cluster_name: skywalking
+                  timeout: 0.250s
+                client_config:
+                  service_name: front-envoy
+                  instance_name: front-envoy-1
+          codec_type: auto
+          stat_prefix: ingress_http
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: backend
+              domains:
+              - "*"
+              routes:
+              - match:
+                  prefix: "/"
+                route:
+                  cluster: service1
+                decorator:
+                  operation: checkAvailability
+          http_filters:
+          - name: envoy.filters.http.router
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
+              start_child_span: true
+  clusters:
+  - name: service1
+    connect_timeout: 0.250s
+    type: strict_dns
+    lb_policy: round_robin
+    http2_protocol_options: {}
+    load_assignment:
+      cluster_name: service1
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: service1
+                port_value: 8000
+  - name: skywalking
+    connect_timeout: 1s
+    type: strict_dns
+    lb_policy: round_robin
+    http2_protocol_options: {}
+    load_assignment:
+      cluster_name: skywalking
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: skywalking-oap
+                port_value: 11800
+admin:
```

Comment:
im wondering if the admin interface is necessary if nothing is accessing it

Reply:
In fact it is useful. After starting the sandbox, the admin port may be used to modify the log level or obtain `stats`.

Reply:
are there any skywalking specific stats ?

Reply:
Yes, skywallking tracer provides some simple stats now. According to the feedback in specific practice, it may continue to be enriched in the future.

Reply:
in which case we should add some small docs about what is added, and a test in `verify.sh`

Reply:
@wbpcode feel free to remove this and i am happy to add some docs/tests against admin after (i might need to ping to confirm what im doing etc)

Reply:
Well, I've added some simple docs and also a test. But I think you can refine it later.

Reply:
ah great, thanks - i havent  run the sandbox again yet - ill do it shortly